locales/en.json

```
@@ -402,6 +402,8 @@
     "notifications_settings": "Notifications",
     "preferences": {
       "enable_autoplay": "Enable Autoplay",
+      "enable_data_saving": "Enable data saving",
+      "enable_data_saving_description": "Save data by preventing attachments from automatically loading. Turns off autoplay if enabled.",
```

I think we don't need to clarify this, it can be inferred by the user
```suggestion
      "enable_data_saving_description": "Save data by preventing attachments from automatically loading.",
```